### ISSUE TITLE
About feature map visualization

### ISSUE THREAD

**Issue by nicolas-dufour** [NVlabs/stylegan3#8]
Hi,
If I understand right from reading the code, to visualize the feature maps and get this coordinate feature map visualization, you only pick the first 3 channels of the feature-map and assign rgb values? No dimension reduction or transformation of the feature map is involved? 
Thanks

**Comment by mshunshin** [comment]
Oh - and as a tip for anyone who wants to extract the various feature layers you have to register a hook to copy them out during a forward pass.


    activation = {}
    def get_activation(name):
        def hook(model, input, output):
            activation[name] = output.detach()
        return hook

Then e.g:
    G.synthesis.L11_276_362.register_forward_hook(get_activation('L11_276_362'))

Then use the forward pass code...

Then you can do:

feature_map = activation['L11_276_362'].to(torch.float) # the intermediate layers might be half precision, so best to convert it back to full.
